recipe/meta.yaml

```
@@ -32,13 +31,15 @@ requirements:
     # added scipy from requirements.txt because it seems to be used in production code in tf_keras/preprocessing/image.py
     # and is not explicitly listed as an optional dependency
     - scipy
-    - tensorflow >=2.17,<2.18
+    - tensorflow >=2.18.0
```

```suggestion
    - tensorflow >=2.18.0,<2.19.0
```
It looks like the upper bound is still required https://github.com/keras-team/tf-keras/blob/v2.18.0/oss_setup.py#L55-L57